doc/man/8/ceph-kvstore-tool.rst

```diff
@@ -15,46 +15,57 @@ Synopsis
 Description
 ===========
 
-:program:`ceph-kvstore-tool` is a kvstore manipulation tool. It allows users to manipulate
-leveldb/rocksdb's data (like OSD's omap) offline.
+:program:`ceph-kvstore-tool` is a tool to interact with key-value database used internally by ceph.
+Key-value databases are part of object store instances.
+Modes *leveldb* and *rocksdb* are for FileStore and *bluestore-kv* is for BlueStore.
+FileStore uses DB only for OMAP, while BlueStore also preserves internal metadata in DB.
+
+OSD must be offline to manipulate on its key-value DB.
+Operations are not logically validated. Modifications of BlueStore's metadata are unadvised.
+
+Column Families
```

Comment:
is there already a command that would let us list the columns?

Reply:
@jdurgin There was not. Now it is and is called "list-columns"